apps/gauzy/src/app/pages/dashboard/accounting/accounting.component.scss

```
@@ -93,9 +93,12 @@
     padding-bottom: 0;
   }
 
+  // `unset, auto`: the dashboard page owns the single scrollbar now (see
```

Stylelint (17.14.1)

[error] Unknown word `unset, (CssSyntaxError)

(CssSyntaxError)